dev-build/conf/routes

```diff
@@ -54,21 +54,6 @@ OPTIONS    /email
 # Business data
 GET            /business-data/stocks.json                                                                                        controllers.StocksController.stocks
 
-# Weather - EVEN OLDER
-GET            /weather/city/:id.json                                                                                            weather.controllers.WeatherController.forCity(id)
-GET            /weather/city.json                                                                                                weather.controllers.LocationsController.whatIsMyCity()
-GET            /weather/locations                                                                                                weather.controllers.LocationsController.findCity(query: String)
-GET            /weather/forecast/:id.json                                                                                        weather.controllers.WeatherController.forecastForCityId(id)
-
-# Weather - OLD
-GET            /weatherapi/city/:id.json                                                                                         weather.controllers.WeatherController.forCity(id)
-GET            /weatherapi/city.json                                                                                             weather.controllers.LocationsController.whatIsMyCity()
-GET            /weatherapi/locations                                                                                             weather.controllers.LocationsController.findCity(query: String)
-GET            /weatherapi/forecast/:id.json                                                                                     weather.controllers.WeatherController.forecastForCityId(id)
-
-#Weather - DCR
-GET            /weather                                                                                                          weather.controllers.WeatherController.theWeather()
```

Comment:
Are we going to delete the endpoints from the router as well? https://github.com/guardian/platform/blob/027bcaecaf0ad92f4b6689b1ac8a710973325df5/router/files/router.conf#L393

Reply:
Yes I think we should. Thanks! I'll create a PR in that repo